warehouse/accounts/views.py

```diff
@@ -1069,13 +1089,13 @@ def _error(message):
     request.db.add(
         JournalEntry(
             name=project.name,
-            action=f"accepted {desired_role} {user.username}",
+            action=f"add {desired_role} {user.username}",
```

Comment:
```suggestion
            action=f"accepted {desired_role} {user.username}",
```

We should leave the legacy Journals as they are.